src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3176,6 +3176,19 @@ operand immL_32bits()
   interface(CONST_INTER);
 %}
 
+// Int Immediate: power of 2, positive
```

Comment:
The comment is misleading: it's not a 2^n, but 2^n-1.

Reply:
👍 - Agreed, this is more descriptive.